frontend/public/components/workload-table.tsx

```diff
@@ -18,36 +19,53 @@ import {
   Selector,
 } from './utils';
 
+const tableColumnClassesWithPhase = [
```

Comment:
We shouldn't be changing the generic workload table header. Either we have a specific implementation for replication controller or we use the default workload table header. This component is too general to have specific knowledge of replication controller phases.